extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -268,11 +288,11 @@ export class AmpAutocomplete extends AMP.BaseElement {
     let filteredData = data.filter(item => {
       switch (this.filter_) {
         case FilterType.SUBSTRING:
-          return includes(item, input);
+          return includes(item.toLowerCase(), input);
```

Comment:
Nit: should we store this in a const before the `switch` statement? It'd make it less likely for a future developer to forget about it, and that would be a cool place to add a comment explaining that it's needed to make the search case insensitive